Recommendation.java

```diff
@@ -23,6 +23,12 @@ public Recommendation()
       imgLink = "https://fl-1.cdn.flockler.com/embed/no-image.svg"
 
   }
+  
+  public Author()
```

Comment:
This is the Recommendation class, so there should ONLY be a constructor for a Recommendation object.

Do NOT create a constructor for Author.

Reply:
In other words, delete lines 27-31
  public Author()
  {
    author = "John Doe";

  }